index.js

```diff
@@ -1,17 +1,81 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    //Question to teacher: Why we cannot write the next two code lines in the same line? (this.items.push(item).sort((a,b) => a-b))
+    this.items.push(item);
+    this.items.sort((a,b) => a-b);
 
-  get(pos) {}
+    this.length ++;
```

Comment:
Aqui en cambio, tienen ganas de abrazarse XD
```suggestion
    this.length++;
```